tidy3d/components/data.py

```diff
@@ -49,7 +49,33 @@ def abs(self):
 """ Base data classes """
 
 
-class Tidy3dData(Tidy3dBaseModel):
+class Tidy3dBaseDataModel(Tidy3dBaseModel):
+    """Tidy3dBaseModel, but with yaml and json IO methods disabled."""
+
+    def _json_string(self, include_unset: bool = True) -> str:
+        """Disable exporting to json string."""
+        raise DataError("Can't export json string of Tidy3dData.")
+
+    def to_json(self, fname: str) -> None:
+        """Disable exporting to json file."""
+        raise DataError("Can't export json file of Tidy3dData.")
```

Comment:
Would it be helpful to add to these errors something like "use `to_file()` instead" or similar?

Reply:
yea, good point, I added a more useful message to these.